spec/2018/01_chronal_calibration_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "../../solutions/2018/01_chronal_calibration"
```

Comment:
I'd put this, and the `input_file_path` stuff in a `spec_helper.rb` file that makes testing easier - you should be able to get away with just `require` instead of `require_relative`